packages/forma-36-react-components/src/components/Card/AssetCard/AssetCard.css

```diff
@@ -44,6 +44,8 @@
   padding: 0 calc(1rem * (14 / var(--font-base-default)));
   height: calc(1rem * (45 / var(--font-base-default)));
   display: flex;
+  justify-content: flex-end;
+  gap: .5rem;
```

Comment:
It doesn't look like `gap` is working on safari, I'd create a separate class for icon similar to `AssetCard__actions`, and use `margin-right` with tokens

<img width="180" alt="Screenshot 2020-11-06 at 14 46 01" src="https://user-images.githubusercontent.com/6163988/98372941-cd56a180-203e-11eb-8c90-605186e45e16.png">